src/app/features/approver/approver.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Box, Circle, Flex } from 'leather-styles/jsx';
+
+import { Button } from '@app/ui/components/button/button';
+import { Callout } from '@app/ui/components/callout/callout';
+import { Flag } from '@app/ui/components/flag/flag';
+import { ItemInteractive } from '@app/ui/components/item/item-interactive';
+import { ItemLayout } from '@app/ui/components/item/item.layout';
+
+import { Approver } from './approver';
+
+const meta: Meta<typeof Approver> = {
+  component: Approver,
+  tags: ['autodocs'],
+  title: 'Feature/Approver',
+
+  render: ({ children, ...args }) => (
+    <Flex maxW="390px" h="680px" border="1px solid lightgrey" overflowY="auto">
```

Comment:
Mark has told me to make the popup height `600px`. Is that good with you? Or do you think we should have it variable based on the popup type? 

I found that `popup.ts` accepts dimensions but we don't use them